Git-Collabration-Example/ViewController.swift

```diff
@@ -6,13 +6,40 @@
 //
 
 import UIKit
+import SafariServices
 
 class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+	title = "Collaboration"
         // Do any additional setup after loading the view.
+
+	//this is for abm04 nvigation and safari test
+    }
+
+override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        print("viewWillAppear")
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        print("s viewDidAppear")
```

Comment:
Were you able to get any result with the print value, otherwise make changes

Reply:
its only for console, practice purpose only.